src/ipyradiant/query/__init__.py

```diff
@@ -3,7 +3,8 @@
 # Copyright (c) 2021 ipyradiant contributors.
 # Distributed under the terms of the Modified BSD License.
 
-__all__ = ["QueryWidget", "service_patch_rdflib"]
+__all__ = ["LegacyQueryWidget", "QueryWidget", "service_patch_rdflib"]
```

Comment:
Leaving old query (LegacyQueryWidget) in the library in case we want to use parts from it later (however unlikely)